doc/ansible/dev/developer_guide.md

```diff
@@ -220,7 +220,29 @@ developer can also do:
 $ operator-sdk new --type ansible --kind Foo --api-version foo.example.com/v1alpha1 foo-operator --generate-playbook
 ```
 
-## Testing an Ansible operator locally
+### Custom Resource file
+
+The Custom Resource file format is Kubernetes resource file. The object has
+mandatory fields:
+
+**apiVersion**:  The version of the Custom Resource that will be created.
+
+**kind**:  The kind of the Custom Resource that will be created
+
+**metadata**:  Kubernetes specific metadata to be created
+
+**spec**:  This is the key-value list of variables which are passed to Ansible.
+This field is optional and will be empty by default.
+
+**annotations**: Kubernetes specific annotations to be appened to the CR. See
+the below section for Ansible Operator specifc annotations.
+
+#### Ansible Operator annotations
+This is the list of CR annotations which will modify the behavior of the operator:
+
+**ansible.operator-sdk/reconcile-period**: Used to specify the reconciliation interval for the CR.
```

Comment:
I would add a little info on how the value is parsed. It uses go's standard library to parse string representations like "3m" or "20s". It would be good to make that clear somehow, either by referencing a standard, or just referencing the go docs. As specific question that may be worth clarifying: what happens if I omit a unit? Does it assume seconds?